knowledge-base/actions/cla-assistant/github-action/action-security.yml

```diff
@@ -0,0 +1,14 @@
+name: "Handling CLAs and DCOs via GitHub Action"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
```

Comment:
Please review the permissions, I think there is only `pull-request: read` and `issues: write` permissions.
and also assign proper reasons to them.